src/backend/vulkan/src/lib.rs

```diff
@@ -510,6 +520,7 @@ impl hal::Instance<Backend> for Instance {
                     handle: device,
                     extensions,
                     properties,
+                    get_physical_device_properties: self.get_physical_device_properties.clone(),
```

Comment:
since it's needed in multiple places, let's just put it into `RawInstance` struct, alongside the debug messenger and stuff